ui/components/component-library/button-link/README.mdx

```diff
@@ -101,6 +101,24 @@ import { ButtonLink } from '../../component-library';
 <ButtonLink href="/">Href example</ButtonLink>;
 ```
 
+### External Link
+
+When an `externalLink` prop is passed it adds the `target="_blank"` and `rel="noopener noreferrer"` attributes. 
+
+`rel="noreferrer noopener"` is used in links to prevent security vulnerabilities that can be exploited by malicious websites. It disables the window.opener property and prevents the new page from sending the referrer information, providing an additional layer of security.
+
+<Canvas>
+  <Story id="components-componentlibrary-buttonlink--href" />
+</Canvas>
+
+```jsx
+import { ButtonLink } from '../../component-library';
+
+<ButtonLink href="https://metamask.io" externalLink>
+  Anchor element with external link
+</ButtonLink>;
```

Comment:
Just check this code example reflects the story but I think it does so not updates here